scripts/new-config.sh

```diff
@@ -12,6 +12,9 @@ fi
 
 CONFIG_NAME=$1
 
+BASEDIR=$(pwd)/$(dirname "$0")
```

Comment:
Is the intention to always create the examples in `scripts/../examples/local`?

Reply:
So far yes.

Reply:
See this part of the Readme ```To create your own examples or test custom configurations, use the bin/new-config.sh script. It accepts the name of your example as the only argument, and it will generate a config template in the un-tracked examples/local/ directory.```